### PR TITLE
Line artifact fixes

### LINKED ISSUE #3: Vertical line apperes  in ios device but working perfectly on Android
IMG_3251

A vertical line appears in the ios device but working perfectly on Android.

### PR DESCRIPTION
**Pull request by sqrlmstr5000**
# Description

Implemented fixes for line artifact mentioned in #3 and #73. These changes fix the issue in my testing. 

Fixes #3 and #73

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Self Code Review
- [ ] Tested on example project

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Changed the example project to contain the new feature (only if applies)

### CODE REVIEW

File: lib/flip_card.dart [hunk 1]
```diff
@@ -37,7 +37,8 @@ class AnimationCard extends StatelessWidget {
         }
         return Transform(
           transform: transform,
-          alignment: Alignment.center,
+          filterQuality: FilterQuality.none,
```
Comment (ciriousjoker) [inline lib/flip_card.dart:40]: Does this have any performance implications?
Given the [documentation](https://api.flutter.dev/flutter/widgets/Transform/filterQuality.html), it should be faster?

File: lib/flip_card.dart [hunk 2]
```diff
@@ -37,7 +37,8 @@ class AnimationCard extends StatelessWidget {
...
+          alignment: FractionalOffset.center,
```
Comment (ciriousjoker) [inline lib/flip_card.dart:41]: Why is this necessary?
As per [documentation](https://api.flutter.dev/flutter/painting/FractionalOffset-class.html), the information described by this is the same.

**Comment by ciriousjoker** [comment]
@tsquillario Thanks for your contribution! The change seems simple and should work out of the box. I'm just a bit curious about why it works.

Not sure when I get around to testing it (so I can merge it), but hopefully next week.

**Comment by sqrlmstr5000** [comment]
I'm not really sure why it works unfortunately. Once I made these two changes it fixed the issue. 

I have a repo in this comment to reproduce the issue. 
https://github.com/BrunoJurkovic/flip_card/issues/73#issuecomment-1470128656

**Comment by ciriousjoker** [comment]
According to [this comment](https://github.com/BrunoJurkovic/flip_card/issues/3#issuecomment-1253397743):

```dart
return Transform(
  transform: transform,
  filterQuality: FilterQuality.none,
  alignment: Alignment.center,
  child: child,
);
```

The `Alignment.center` still works. Can you try to reproduce the issue with and without the change to the alignment to see if it's necessary?

**Comment by sqrlmstr5000** [comment]
It seems inconsistent. I fired up the test repo and the line didn't show at first until I tried to drag and drop one of the cards. Now the line appears. Seems to only happen on tablet as well. Something about when the cards are redrawn. The test repo is using the release `flip_card: ^0.7.0` no modifications to the package code. 

Screenshot_1680010085

**Comment by ciriousjoker** [comment]
Ok, so both of the properties are necessary to produce 100% working solution? And: If you remove one of then, the problem comes back in some random cases?

**Comment by ciriousjoker** [comment]
@tsquillario reminder to please check if both properties are necessary to guarantee that the bug is fixed.

As I said, I cannot verify the bug myself so I can't check which of these are required to fix it.

I'll remind you every now and then. If you want me to stop, just ping me.

**Comment by ciriousjoker** [comment]
I did some testing, here's the result:

Differences in Filterquality:

| none | low | high |
|--------|--------|--------|
| CleanShot 2023-04-19 at 19 14 44@2x | CleanShot 2023-04-19 at 19 18 58@2x | CleanShot 2023-04-19 at 19 15 52@2x | 

Given the speed, this is probably irrelevant. If anyone complains, we can just offer it as a parameter.

`FractionalOffset.center` seems to have no effect whatsoever.

I'll therefore merge this.

**Review by ciriousjoker — APPROVED** [review]
lgtm, risk of breaking something is low and would be easily fixable

**Comment by ciriousjoker** [comment]
@tsquillario Thanks for your contribution!

**Pull request merged**